website/package.json

```diff
@@ -8,7 +8,8 @@
     "version": "docusaurus-version",
     "rename-version": "docusaurus-rename-version",
     "crowdin-upload": "crowdin --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin --config ../crowdin.yaml download -b master"
+    "crowdin-download": "crowdin --config ../crowdin.yaml download -b master",
+    "check-links": "yarn blc http://localhost:3000 -ro --exclude \"http://localhost:3000/docs/ru\" --exclude \"http://localhost:3000/docs/zh-CN\" --exclude \"https://playground.substrate.dev/\" --exclude \"https://player.twitch.tv/?channel=paritylivecoding\" --exclude \"http://localhost:8000/\""
```

Comment:
```
$ yarn check-links

yarn run v1.22.4
$ yarn blc http://localhost:3000 -ro --exclude "http://localhost:3000/docs/ru" --exclude "http://localhost:3000/docs/zh-CN" --exclude "https://playground.substrate.dev/" --exclude "https://player.twitch.tv/?channel=paritylivecoding" --exclude "http://localhost:8000/"
$ /home/joshy/ProgrammingProjects/substrate-developer-hub.github.io/website/node_modules/.bin/blc http://localhost:3000 -ro --exclude http://localhost:3000/docs/ru --exclude http://localhost:3000/docs/zh-CN --exclude https://playground.substrate.dev/ --exclude 'https://player.twitch.tv/?channel=paritylivecoding' --exclude http://localhost:8000/
Error: Unhandled Rejection. Error: connect ECONNREFUSED 127.0.0.1:3000
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

It also seems funny to use a different link check here than in CI. I was probably too hasty installing one in CI without discussing it first. What if we add this exact same command to the CI instead of the one I added yesterday?

Reply:
Ah, nevermind. I see the CI was updated to blc.